src/js/components/modals/AppModalComponent.jsx

```diff
@@ -98,7 +99,12 @@ var AppModalComponent = React.createClass({
     // All status below 300 are actually not an error
     if (status < 300) {
       this.onCreateApp();
-
+    } else if (status === 409) {
+      this.setState({
+        responseErrorMessages: {general: `Failed to change and deploy the new
```

Comment:
I like the message. Maybe you could bring out the reason of the failed deployment a little bit more, not sure. @pierlo-upitup ?
```Applying of new configuration is blocked by an existing deployment. Press the button again to force the operation.```

Please put the message into this file:
https://github.com/mesosphere/marathon-ui/blob/master/src/js/constants/AppFormErrorMessages.js#L33
And access it with ```AppFormErrorMessages.general[4]```.